hz: 10
dist_to_local_goal: 1.5
bin_num: 3
bin_step_degree: 25
